build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+TAG=$(git rev-parse --short HEAD)
+
+docker build -t quay.io/eclipse/che-cert-manager-ca-cert-generator:${TAG} .
```

Comment:
You may provide a .github action as well to automatically build and publish the image on every commit and check build on PR ?

Reply:
Sound interesting! But I would like to have it in a separate PR.